### ISSUE TITLE
Oracle does not support IF NOT EXISTS on CREATE SEQUENCE

### ISSUE THREAD

**Issue by saulotoledo** [sqlfluff/sqlfluff#8439]
### Search before asking

- [x] I searched the [issues](https://github.com/sqlfluff/sqlfluff/issues) and found no similar issues.

### What Happened

Starting at Oracle 19c RU 19.28, `CREATE SEQUENCE` accepts `IF NOT EXISTS`. The current version of SQLFluff does not support it.

### Expected Behaviour

We should be able to parse it.

### Observed Behaviour

The parser fails.

### How to reproduce

1. Create a `repro.sql` file:

```sql
CREATE SEQUENCE IF NOT EXISTS MY_SEQ ORDER NOCACHE;
```

2. Run `sqlfluff parse repro.sql`

### Dialect

oracle

### Version

4.3.0

### Configuration

Default.

### Are you willing to work on and submit a PR to address the issue?

- [x] Yes I am willing to submit a PR!

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/sqlfluff/sqlfluff/blob/main/CODE_OF_CONDUCT.md)

**Comment by saulotoledo** [comment]
Hi @peterbud, do you have a suggestion to make this version-dependent? Or should we just add it to the grammar?

**Comment by peterbud** [comment]
> Hi [@peterbud](https://github.com/peterbud), do you have a suggestion to make this version-dependent? Or should we just add it to the grammar?

No need to be version dependent. Just add it to the grammar. Out goal is not to be a syntax checker, just linting.

**Comment by saulotoledo** [comment]
@peterbud This seems to be an issue with `CREATE INDEX` as well: https://docs.oracle.com/en/database/oracle/oracle-database/19/sqlrf/CREATE-INDEX.html#if-not-exists

I will check this carefully soon (maybe tomorrow) and consider fixing both then.

**Comment by saulotoledo** [comment]
Ops, I saw the linked PR and created another task issue for the index one: https://github.com/sqlfluff/sqlfluff/issues/8443
